### DISCUSSION TITLE
Issue with finding word by my own template

### DISCUSSION THREAD

**Discussion by rawezh80** [projectdiscovery/nuclei-templates#7702] [Q&A] [answered]
when i run nuclei option `-l`

```
rawezh@Rawezh:/mnt/c/Users/rawez/Desktop/ppp$ nuclei -l bb1.txt -t own.yaml

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v2.9.8

                projectdiscovery.io

                projectdiscovery.io
[INF] Current nuclei version: v2.9.8 (latest)
[INF] Current nuclei-templates version: v9.5.7 (latest)
[INF] New templates added in latest release: 21
[INF] Templates loaded for current scan: 1
[INF] Targets loaded for current scan: 3101
[basic-example] [http] [info] https://admin.instacart.com
[basic-example] [http] [info] https://beta.instacart.com
[basic-example] [http] [info] https://c4.kayak.com
[basic-example] [http] [info] https://community.zomato.com
[basic-example] [http] [info] https://content.superawesome.com
[basic-example] [http] [info] https://de.trustpilot.com
[basic-example] [http] [info] https://emoji.tinder.com
[basic-example] [http] [info] https://food.grab.com
[basic-example] [http] [info] https://fr.legal.trustpilot.com
[basic-example] [http] [info] https://freight.indrive.com
[basic-example] [http] [info] https://friends.figma.com
[basic-example] [http] [info] https://giant-tiger.pbis-cf.instacart.com
[basic-example] [http] [info] https://ie.trustpilot.com
[basic-example] [http] [info] https://ipo.trustpilot.com
[basic-example] [http] [info] https://kan-187653-560272.affiliate.kayak.com
[basic-example] [http] [info] https://legal.trustpilot.com
^Z
[2]+  Stopped                 nuclei -l bb1.txt -t own.yaml
```

when I run nuclei with option `-u`

```
rawezh@Rawezh:/mnt/c/Users/rawez/Desktop/ppp$ nuclei -u https://legal.trustpilot.com -t own.yaml

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v2.9.8

                projectdiscovery.io

[INF] Current nuclei version: v2.9.8 (latest)
[INF] Current nuclei-templates version: v9.5.7 (latest)
[INF] New templates added in latest release: 21
[INF] Templates loaded for current scan: 1
[INF] Targets loaded for current scan: 1
[INF] No results found. Better luck next time!  
```

and this is my template code:

```yaml
id: basic-example

info:
  name: Test HTTP Template
  author: pdteam
  severity: info

http:
  - method: GET
    path:
      - "{{BaseURL}}"
    matchers:
      - type: word
        words:
          - "https://twitter.com"
```

As you can see, I ran the same code, just changed the -l option to -u, but I got two different answers. I'm sure there are no problems with my code please review thanks

**Comment by MetzinAround** [comment]
What are the contents of the  `bb1.txt` file?

**Reply by MetzinAround** [reply]
It looks like bb1.txt it has over 3,000 targets in it, while you only chose 1 target with the `-u` flag. 

Since you're running a scan against 3000 targets in one scan and only 1 in the other, it makes sense that it would have very different results.

**Comment by ehsandeep** [comment]
@Rawezho1o it works for me all the time.

```console
nuclei -u https://legal.trustpilot.com -t test.yaml 

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v2.9.8

		projectdiscovery.io

[INF] Current nuclei version: v2.9.8 (latest)
[INF] Current nuclei-templates version: v9.5.7 (latest)
[INF] New templates added in latest release: 21
[INF] Templates loaded for current scan: 1
[INF] Targets loaded for current scan: 1
[basic-example] [http] [info] https://legal.trustpilot.com
```

**Reply by rawezh80** [reply]
Thanks for your answers but I don't know why I have a problem with a file containing many domains nuclei works very well with it but when I scan only one domain has a problem I use the same template so it should have the same result can you it there any way to fix it please????
rawezh@Rawezh:/mnt/c/Users/rawez/Desktop/ppp$ nuclei -u https://legal.trustpilot.com -t own.yaml

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v2.9.8

                projectdiscovery.io



[INF] Current nuclei version: v2.9.8 (latest)
[INF] Current nuclei-templates version: v9.5.7 (latest)
[INF] New templates added in latest release: 21
[INF] Templates loaded for current scan: 1
[INF] Targets loaded for current scan: 1
[INF] No results found. Better luck next time!

and this is my file 

[bb1.txt](https://github.com/projectdiscovery/nuclei-templates/files/12069623/bb1.txt

**Reply by rawezh80** [reply]
I run it with option -debug and i Got this warning can you tell me how fix it please ????
rawezh@Rawezh:/mnt/c/Users/rawez/Desktop/ppp$ nuclei -u https://legal.trustpilot.com -t own.yaml -debug

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v2.9.8

                projectdiscovery.io

[INF] Current nuclei version: v2.9.8 (latest)
[INF] Current nuclei-templates version: v9.5.7 (latest)
[INF] New templates added in latest release: 21
[INF] Templates loaded for current scan: 1
[INF] Targets loaded for current scan: 1
[INF] [basic-example] Dumped HTTP request for https://legal.trustpilot.com

GET / HTTP/1.1
Host: legal.trustpilot.com
User-Agent: Mozilla/5.0 (Windows NT 6.3; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2225.0 Safari/537.36
Connection: close
Accept: */*
Accept-Language: en
Accept-Encoding: gzip

[WRN] [basic-example] Could not execute request for https://legal.trustpilot.com: context deadline exceeded
[INF] No results found. Better luck next time!
rawezh@Rawezh:/mnt/c/Users/rawez/Desktop/ppp$

**Comment by MetzinAround** [comment, marked as answer]
Try changing the `-timeout` flag to a higher amount to avoid the context deadline.

**Reply by rawezh80** [reply]
thanks a lot just by set flag -timeout my issue fixed
no meter what rang of timeout 
rawezh@Rawezh:/mnt/c/Users/rawez/Desktop/ppp$ nuclei -u https://legal.trustpilot.com -t own.yaml  -timeout 1

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v2.9.8

                projectdiscovery.io

[INF] Current nuclei version: v2.9.8 (latest)
[INF] Current nuclei-templates version: v9.5.7 (latest)
[INF] New templates added in latest release: 21
[INF] Templates loaded for current scan: 1
[INF] Targets loaded for current scan: 1
[basic-example] [http] [info] https://legal.trustpilot.com
